scipy/linalg/decomp.py

```diff
@@ -778,7 +778,7 @@ def eigvals_banded(a_band, lower=False, overwrite_a_band=False,
         The bands of the M by M matrix a.
     lower : boolean
         Is the matrix in the lower form. (Default is upper form)
-    overwrite_a_band:
+    overwrite_a_band : boolean
```

Comment:
I fixed up a lot more of these https://github.com/rmcgibbo/scipy/commit/6e71cf0baba66d3df0e378ad41b1311b96b33c64